packages/sdk/src/index.ts

```diff
@@ -56,8 +56,14 @@ export {
   TriggerPatterns,
   TeamsFxAdaptiveCardActionHandler,
   TeamsFxBotCommandHandler,
+  TeamsFxBotSsoCommandHandler,
+  BotSsoExecutionActivityHandler,
+  BotSsoConfig,
+  BotSsoExecutionDialogHandler,
 } from "./conversation/interface";
 export { ConversationBot } from "./conversation/conversation";
+export { BotSsoExecutionDialog } from "./conversation/sso/botSsoExecutionDialog";
```

Comment:
I'm not sure whether we want to export BotSsoExecutionDialog and DefaultBotSsoExecutionActivityHandler. Can you share how they serves for the feature? My major concern is they're our internal implementation, if we exported them, we need to consider more when we want to ajust our internal implementations.

Reply:
Yes, we will export BotSsoExecutionDialog, since it is a general dialog and user can use it process custom command as below:
```
    this.ssoExecutionDialog?.addCommand(
      async (
        context: TurnContext,
        tokenResponse: TeamsBotSsoPromptTokenResponse,
        message: CommandMessage
      ) => {
        ...
        user's logic
        ...
      },
      triggerPattern
    );
```

For DefaultBotSsoExecutionActivityHandler, we can hide and do not export it